examples/with-eth-passkeys-galore/src/pages/index.tsx

```diff
@@ -366,6 +560,53 @@ export default function Home() {
           </form>
         </div>
       )}
+      {wallet && (
+        <div>
+          <h2>... and now let&apos;s sign a transaction!</h2>
```

Comment:
maybe a little more readable
```suggestion
          <h2>... and now let{"'"}s sign a transaction!</h2>
```

or even 
```
          <h2>... and now {"let's"} sign a transaction!</h2>
```